frontends/p4/inlining.cpp

```diff
@@ -500,9 +500,20 @@ const IR::Node* GeneralInliner::preorder(IR::P4Control* caller) {
             // Use temporaries for these parameters
             std::set<const IR::Parameter*> useTemporary;
 
-            auto call = workToDo->uniqueCaller(inst);
+            const IR::MethodCallStatement *call = nullptr;
+            for (auto m : workToDo->callToInstance) {
+                if (m.second != inst) continue;
+                if (call) {
+                    if (!call->equiv(*m.first)) {
```

Comment:
is this precise enough to distinguish between two identifiers with the same name but different scopes?

Reply:
Can't happen as the previous UniqueNames pass will have renamed them to be different.